### PR TITLE
Update cloud config endpoint routing

### PR DESCRIPTION
**Pull request by Sean-Holcomb**
## What does this PR change?
This PR updates the routing to allow for configuration endpoints on the cloud cost container

## Does this PR rely on any other PRs?
https://github.com/kubecost/kubecost-cost-model/pull/2187

### CODE REVIEW

File: cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml [hunk 1]
```diff
@@ -969,25 +969,33 @@ data:
             proxy_set_header  X-Real-IP  $remote_addr;
             proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
         }
-        location = /model/cloudCost/integration/export {
+        location = /model/cloud/config/export {
```
Comment (ameijer) [inline cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml]: so was the FE broken? or will they need to impl these new paths?

File: cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml [hunk 2]
```diff
@@ -969,25 +969,33 @@ data:
...
-            proxy_pass http://cloudCost/cloudCost/integration/disable;
+            proxy_pass http://cloudCost/cloud/config/disable;
+            proxy_redirect off;
+            proxy_set_header Connection "";
+            proxy_set_header  X-Real-IP  $remote_addr;
+            proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+        }
+        location = /model/cloud/config {
+            proxy_read_timeout          {{ .Values.kubecostFrontend.timeoutSeconds | default 300 }};
+            proxy_pass http://cloudCost/cloud/condig;
```
Comment (ameijer) [inline cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml]:
```suggestion
            proxy_pass http://cloudCost/cloud/config;
```

File: cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml (hunk 1 above)
Comment (Sean-Holcomb) [reply cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml]: Its unclear to me that they ended up using this endpoint

Comment (ameijer) [reply cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml]: do they use any of these endpoints you modified?

Comment (Sean-Holcomb) [reply cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml]: just confirmed that the modified endpoints are not currently used

**Review by nik-kc — APPROVED** [review]

**Pull request merged**